static-assets/themes/cstudioTheme/css/console.css

```diff
@@ -69,9 +69,8 @@ input[type="checkbox"].content-type-selector--checkbox {
 }
 
 .work-area-empty, .work-area-error {
-  height: 100vh;
+  height: calc(100vh - 55px);
   display: flex;
-  margin-top: 50px;
```

Comment:
How does this work in the case of nothing selected with the top bar present?

Reply:
It works good, actually in embedded mode it doesn't take the full height, will add the styles for embedded